### ISSUE TITLE
Modifications to ScheduleEvent properties (Edinburgh Roadmap Item)

### ISSUE THREAD

**Issue by tsconn23** [edgexfoundry/edgex-go#796]
As discussed during the Device Services and Core Working Group meetings, we will be changing the property signature of ScheduleEvents as follows:

`pkg/models/scheduleevent.go`
- Add "Owner" property to contain the name of the service that executes the event as per schedule
- Change "Service" property to "Target". Service is too overloaded a word, and the usage of the property is to indicate which service needs to be targeted when invoking the scheduled event's endpoint.

Three follow-up issues once this is done:
- Change queries via metadata to obtain scheduled events by Owner instead of Name.
- Change device service SDKs (C/Go) to populate the Owner property and to utilize renamed Target property where applicable.
- Change support-scheduler to utilize renamed Target property where applicable.

**Comment by xmlviking** [comment]
Recent conversations during the Core Working group and others have revealed that ScheduleEvent will no longer require an Owner field.   With the movement of the scheduler and scheduleEvent logic moving from Core-Metadata into the Support-Scheduler proper we no longer need this distinction.  Additionally the devices services working group has discussed the renaming or removal of the schedule and scheduler event types for it's requirements.  

While the refactoring of the support-scheduler is underway the pkg/models/  will not be used by the refactored support-scheduler.  When device services has renamed or removed the required types in the pkg/models directory, the support-scheduler wll move those types back into common.   

As part of the refactoring for support-scheduler the current pkg/models/addressable.go will no longer be used in the support-scheduler.   this type will be folded into the new scheduleEvent.go struct currently residing in the support-scheduler models directory.  As mentioned previously this will eventually be migrated out of the support-scheduler and placed in the pkg/models directory..

Renaming "Service" to "Target" will still take place as part of this fix.

**Comment by tsconn23** [comment]
These changes were incorporated into the new Interval / IntervalAction types. Closing.
